src/Hydrators/Family.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Choredo\Hydrators;
+
+use Assert\Assert;
+use Choredo\Entities;
+use Ramsey\Uuid\Uuid;
+use const Choredo\DAYS_OF_WEEK;
+use const Choredo\SHORT_DATA_FIELD_MAX_SIZE;
+
+class Family
```

Comment:
Let's suffix these classes with their type to avoid multiple `Family` classes, to avoid confusion.

Reply:
:+1: